File: Python/Tarifa.py
class Tarifa:
    _id_contador = 0

    def __init__(self, valor, tipo, unidad):
        Tarifa._id_contador+=1
        self.id = Tarifa._id_contador
        self.valor = valor
        self.tipo = tipo
        self.unidad = unidad

    #Metodos Getters
    def get_id(self):
        return self.id
    
    def get_valor(self):
        return self.valor
    
    def get_tipo(self):
        return self.tipo
    
    def get_unidad(self):
        return self.unidad
    
    #Metodos Setters
    def set_valor(self, valor):
        self.valor = valor

    def set_tipo(self, tipo):
        self.tipo=tipo
    
    def set_unidad(self, unidad):
        self.unidad = unidad

File: Java/Tarifa.java
public class Tarifa {
    private int id;
    private static int contadorId;
    private double valor;
    private String tipo;
    private int unidad;

    public Tarifa(double valor, String tipo, int unidad) {
        this.id = ++Tarifa.contadorId;
        this.valor = valor;
        this.tipo = tipo;
        this.unidad = unidad;
    }
    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public static int getContadorId() {
        return contadorId;
    }
    public static void setContadorId(int contadorId) {
        Tarifa.contadorId = contadorId;
    }
    public double getValor() {
        return valor;
    }
    public void setValor(double valor) {
        this.valor = valor;
    }
    public String getTipo() {
        return tipo;
    }
    public void setTipo(String tipo) {
        this.tipo = tipo;
    }
    public int getUnidad() {
        return unidad;
    }
    public void setUnidad(int unidad) {
        this.unidad = unidad;
    }

    

    
}
